coretex/entities/project/project.py

```diff
@@ -33,7 +35,7 @@ class Project(BaseObject):
     tasks: List[Task]
 
     @classmethod
-    def createProject(cls, name: str, projectType: ProjectType, description: Optional[str] = None) -> Optional[Self]:
+    def createProject(cls, name: str, projectType: ProjectType, description: Optional[str] = None) -> Self:
```

Comment:
Update documentation as function signature changed and it is now raising an exception